src/storekit.cs

```diff
@@ -132,34 +140,39 @@ interface SKMutablePayment {
 		[Availability (Deprecated = Platform.iOS_5_0, Message = "Use 'PaymentWithProduct (SKProduct)' after fetching the list of available products from 'SKProductRequest' instead.")]
 		SKMutablePayment PaymentWithProduct (string identifier);
 
+		[Watch (6, 2)]
```

Comment:
those should be on the type
because without a [Watch] on the type we'll assume it's been there since watchOS 2.0 (and that's not correct)

Reply:
Ah, got it. I will go through all the APIs and mark them as appropriate :)